src/site/content/en/blog/storage-for-the-web/index.md

```diff
@@ -150,10 +149,9 @@ if (navigator.storage && navigator.storage.estimate) {
 }
 ```
 
-The StorageManager isn't [implemented][caniuse-sm] in all browsers yet, so you
-must feature detect it before using it. Even when it is available, you must
-still catch over-quota errors (see below). In some cases, it's possible for
-the available quota to exceed the actual amount of storage available.
```

Comment:
Do not delete this section, it's still contains critical information.

Reply:
StorageManager may be, but the required API `.estimate()` is not https://caniuse.com/mdn-api_storagemanager_estimate

Reply:
It still could use some updating. Please see the rewrite immediately below this comment (line 151).